libs/blocks/merch-card/merch-card.css

```diff
@@ -0,0 +1,376 @@
+/* stylelint-disable selector-class-pattern */
+/* stylelint-disable-next-line selector-class-pattern */
+@media only screen and (min-width: 680px) {
+  .consonant-CardsGrid--2up {
+      grid-template-columns: repeat(auto-fit,minmax(380px,max-content)) !important;
+  }
+}
+.merch-card.consonant-Card {
+  border-radius: 16px;
+}
+
+.merch-card.segment-blade {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  min-height: 222px;
+  width: 100%;
+  max-width: 100%;
+  flex: 1;
+  min-width: 378px;
+  max-width: 378px;
+}
+.merch-card p {
+  margin-bottom: 8px;
+}
+
+@media only screen and (max-width: 1400px) {
+  .merch-card.segment-blade {
+    min-width: 317px;
+    max-width: 317px;
+  }
+}
```

Comment:
Media queries should live at the bottom of the stylesheet

Reply:
moved